lib/src/modules/instabug.dart

```diff
@@ -412,4 +412,9 @@ class Instabug {
     final darkKey = await dark.obtainKey(configuration);
     return _host.setCustomBrandingImage(lightKey.name, darkKey.name);
   }
+
+  /// This API has to be call when using custom app rating prompt
+  static Future<void> willRedirectToAppStore() async {
```

Comment:
What are your thoughts about renaming it to a more general name that fits all platforms/stores like `willRedirectToStore` or `willRedirectToPlatformStore`

Please share your thoughts @Instabug/cross-platform-team.

Reply:
I second that. `willRedirectToStore` feels more platform-agnostic to me.